docs/CHILI-GraFx/concepts/federated-single-sign-on/oidc/index.md

```diff
@@ -1,21 +1,41 @@
-# Federated Single sign-on: OIDC
+# Federated Single Sign On: OIDC
 
-This document lists the info for your OIDC Identity Providers in the CGX Platform.
+This document lists the steps required for setting up Federated Single Sign On from your OIDC Identity Provider ("IDP") to CHILI GraFx.
 
-## CHILI GraFx will need
+## 1. Configure your IDP
 
-- Issuer URL 
+Create an OIDC application in your IDP
+
+Redirect URI: `https://login.chiligrafx.com/login/callback`
+
+Please configure your IDP to provide at least following claims in the **ID token**:
+
+| **Claim name**                        | **Description**                                           |
+| ------------------------------------- | --------------------------------------------------------- |
+| `sub`                                 | The user ID. Should be set to the email address           |
+| `email`                               | The email address of the user                             |
+| `given_name`                          | The given name of the user                                |
+| `family_name`                         | The family name of the user                               |
+| `https://chili-publish.com/CGXGroups` | A list of UUIDs of the GraFx groups the user should be in |
+
+Please enable both the `Standard` (auth code) and the `Implicit` flows. 
```

Comment:
nitpicking - similar to comment in SAML - auth flows should be along with Redirect URI, prb in table

Reply:
Will do